docs/statistics.rst

```diff
@@ -0,0 +1,51 @@
+Anonymous Statistics
+====================
+
+Tezos Reward Distributor collects anonymous statistics after each payout. These statistics are purely for analytical purposes and potential marketing information.
+
+Nothing that TRD collects can be traced back to a specific delegate or delegator. The goal of the statistics is not to correlate nor discover specific bakeries that are using TRD.
+
+If you wish to opt-out of this collection, start TRD using the `--do_not_publish_stats` option.
+
+Collected Data
+--------------
+
+TRD collects the following statistics after each payout:
+
+* Anonymous identifier
+* Payout cycle
```

Comment:
If you have this information you can trace back the baker....

Reply:
Per the policy, we do not care who the baker is. It would be extremely difficult to trace back to a baker using the cycle and total amount of payments. The stats only show the total amount. The baker could have multiple batches of payments, none of which equal the total. We don't collect any tz1 address information or generate an ID based on their address. We don't collect IP information either.